app/src/main/res/layout-land/fragment_welcome.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="org.fossasia.openevent.general.WelcomeFragment">
+
+    <TextView
+        android:id="@+id/textView"
```

Comment:
Add meaningful variable names

Reply:
elsewhere too

Reply:
Ok. I'll add them

Reply:
@nikit19 the corresponding portrait layout doesn't have variable names for the same. Do I add them only in the landscape layout?

Reply:
If I add ids which aren't there in portrait mode layout, then someone can potentially try using them in Kotlin. Which may be error-prone. The id would exist only in landscape mode and not in portrait mode. If I should add ids, shouldn't I add them for both of them?

Reply:
@nikit19  I've made the changes

Reply:
@nikit19 can you please review my changes?